code/index.html

```diff
@@ -26,10 +26,19 @@ <h1>Does the person have</h1>
         <optgroup label="hair">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+          <option value="white">white hair</option>
+          <option value="orange">orange hair</option>
+          <option value="black">black hair</option>
+          <option value="hidden">hidden hair</option>
+          <option value="purple">purple hair</option>
+          <option value="grey">grey hair</option>
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
         <optgroup label="eyes">
           <option value="hidden">hidden eyes</option>
+          <option value="blue">blue eyes</option>
+          <option value="brown">brown eyes</option>
+          <option value="green">green eyes</option>
           <!-- Add the rest of eye colors as options  -->
         </optgroup>
         <optgroup label="accessories">
```

Comment:
no hat? Probably just missed it.

Reply:
oh yeah! I actually skipped it! I wanted to add but I focused on other things.